ContactDetailsApi/V2/Gateways/Interfaces/ITenureDBGateway.cs

```diff
@@ -0,0 +1,12 @@
+using Hackney.Shared.Tenure.Infrastructure;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ContactDetailsApi.V2.Gateways.Interfaces
+{
+    public interface ITenureDbGateway
+    {
+
+        public Task<List<TenureInformationDb>> GetAllTenures();
```

Comment:
Better to have `<IEnumerable>` in interface definitions rather than `<List>` as its less restrictive in the implementation and a list can be created as late as possible to benefit from deferred execution.